stages/org.osbuild.kickstart

```diff
@@ -382,6 +463,37 @@ def make_autopart(options: Dict) -> str:
     return cmd
 
 
+def make_network(options: Dict) -> List[str]:
+    networks = options.get("network")
+    if networks is None:
+        return []
+    res = []
+    for net in networks:
+        cmd = "network"
+        # simple string options
+        for key in [
+                "device", "bootproto", "onboot", "ip", "ipv6", "gateway",
+                "ipv6gateway", "netmask", "hostname", "essid", "wpakey",
+        ]:
+            if key not in net:
+                continue
+            val = net.get(key)
+            cmd += f" --{key}={val}"
+        # nameservers is a list
+        nameservers = net.get("nameservers")
+        if nameservers:
+            nss = " ".join([f"--nameserver={ns}" for ns in nameservers])
+            cmd += f" {nss}"
+        # activate is special
+        activate = net.get("activate")
+        if activate is True:
```

Comment:
Minor nitpick but comparison to `True` and `False` should be `==`. In this case it won't matter but in other cases it /might/.